resources/qml/Widgets/ComboBox.qml

```diff
@@ -45,7 +46,7 @@ ComboBox
         State
         {
             name: "highlighted"
-            when: (base.hovered || control.hovered) && !control.activeFocus
+            when: ((base != undefined && base.hovered) || control.hovered) && !control.activeFocus
```

Comment:
I think it is a bit weird for such a generic component to check for a somewhat component with `id=base` somewhere up in the component tree. I'm not sure if it will break stuff but why not delete that property check all together?
```suggestion
            when: control.hovered && !control.activeFocus
```

Reply:
Moved this logic out into the components that need it.